contrib/ruby_event_store-outbox/Gemfile

```diff
@@ -13,11 +13,7 @@ gem 'influxdb', '~> 0.8.0', require: false
 gem 'childprocess'
 gem 'rails', ENV['RAILS_VERSION']
```

Comment:
RAILS_VERSION is more, multiple Gemfiles are preferred (each having a lockfile)

Reply:
I will start with one version working and expand later

Reply:
s/ENV['RAILS_VERSION']/6.0.3.4/ then